packages/discord.js-utilities/src/lib/PaginatedMessages/PaginatedMessage.ts

```diff
@@ -404,17 +603,40 @@ export class PaginatedMessage {
 	 * @default "What page would you like to jump to?"
 	 */
 	public static promptMessage = 'What page would you like to jump to?';
+
+	/**
+	 * The messages that are currently being handled by a {@link PaginatedMessage}
+	 * The key is the ID of the message that triggered this {@link PaginatedMessage}
+	 *
+	 * This is to ensure that only 1 {@link PaginatedMessage} can run on a specified message at once.
+	 * This is import when having an editable commands solution.
```

Comment:
This. is. IMPORT.

Reply:
![image](https://user-images.githubusercontent.com/24852502/126035223-b4d89030-41b1-46d6-b6a7-17922da7ff07.png)